code/modules/mob/living/carbon/xenomorph/attack_alien.dm

```diff
@@ -155,49 +157,35 @@
 				"<span class='danger'>Your tackle is blocked by [src]'s shield!</span>", null, 5)
 				return 0
 			M.flick_attack_overlay(src, "disarm")
-			if(knocked_down)
-				if(isYautja(src))
-					if(prob(95))
-						M.visible_message("<span class='danger'>[src] avoids \the [M]'s tackle!</span>", \
-						"<span class='danger'>[src] avoids your attempt to tackle them!</span>", null, 5)
-						playsound(loc, 'sound/weapons/alien_claw_swipe.ogg', 25, 1)
-						return 1
-				else if(prob(80))
-					playsound(loc, 'sound/weapons/alien_claw_swipe.ogg', 25, 1)
-					M.visible_message("<span class='danger'>\The [M] tries to tackle [src], but they are already down!</span>", \
-					"<span class='danger'>You try to tackle [src], but they are already down!</span>", null, 5)
-					return 1
-				playsound(loc, 'sound/weapons/pierce.ogg', 25, 1)
-				KnockDown(rand(M.tacklemin, M.tacklemax)) //Min and max tackle strenght. They are located in individual caste files.
-				M.visible_message("<span class='danger'>\The [M] tackles down [src]!</span>", \
-				"<span class='danger'>You tackle down [src]!</span>", null, 5)
-
-			else
-				var/tackle_bonus = 0
-				if(M.frenzy_aura > 0)
-					tackle_bonus = M.frenzy_aura * 3
-				if(isYautja(src))
-					if(prob((M.tackle_chance + tackle_bonus)*0.2))
-						playsound(loc, 'sound/weapons/alien_knockdown.ogg', 25, 1)
-						KnockDown(rand(M.tacklemin, M.tacklemax))
-						M.visible_message("<span class='danger'>\The [M] tackles down [src]!</span>", \
-						"<span class='danger'>You tackle down [src]!</span>", null, 5)
-						return 1
-					else
-						playsound(loc, 'sound/weapons/alien_claw_swipe.ogg', 25, 1)
-						M.visible_message("<span class='danger'>\The [M] tries to tackle [src]</span>", \
-						"<span class='danger'>You try to tackle [src]</span>", null, 5)
-						return 1
-				else if(prob(M.tackle_chance + tackle_bonus)) //Tackle_chance is now a special var for each caste.
-					playsound(loc, 'sound/weapons/alien_knockdown.ogg', 25, 1)
-					KnockDown(rand(M.tacklemin, M.tacklemax))
-					M.visible_message("<span class='danger'>\The [M] tackles down [src]!</span>", \
-					"<span class='danger'>You tackle down [src]!</span>", null, 5)
-					return 1
 
+			if(knocked_down)
```

Comment:
Being unable to tackle knocked down targets makes for very hard keeping of hosts. You'll might want something like `if(knocked_out || sleeping)`

Reply:
The entire point is to prevent chain stuns.

I've already given the duration a bonus that scales with halloss.

Reply:
It defeats the purpose of the stamina-like damage if it cannot stack. It already has a max value it can reach.

Reply:
Halloss is supposed to stack.

Hard CC is not suppose to stack; stuns are not supposed to chain. This is half the fundamental point of this PR.

Reply:
I understand not stacking `knocked_down`, that is good, but `halloss` should stack, and the xeno should keep adding it to a conscious target if they keep attacking. That's how disabler damage works.

Reply:
You can add a check to see if the target is unconscious and then add the "but they are already down!" message.
If the target is not receiving `knockdown` but is still receiving `halloss` then a message denoting it should be added.